src/Authorization.php

```diff
@@ -72,7 +72,11 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
             }
         }
 
-        return ResponseBridge::fromOAuth2($this->server->getResponse());
```

Comment:
I think it would be better to simply use a `withHeader` call here

```php
return ResponseBridge::fromOAuth2($this->server->getResponse())->withHeader('Content-Type', 'application/json');
```